packages/asset-swapper/src/utils/quote_requestor.ts

```diff
@@ -254,7 +254,12 @@ export class QuoteRequestor {
                 this._warningLogger(result, 'Invalid RFQ-T indicative quote received, filtering out');
                 return false;
             }
-            if (!hasExpectedAddresses([[makerToken, order.makerToken], [takerToken, order.takerToken]])) {
+            if (
+                !hasExpectedAddresses([
+                    [makerToken, order.makerToken],
+                    [takerToken, order.takerToken],
+                ])
+            ) {
```

Comment:
my prettier did this automatically... I wonder if my settings are different than everyone else's?

Reply:
it's failing prettier, run `yarn prettier`